### PR TITLE
fix: [CDS-38222]: descriptive title for dashboard

### PR DESCRIPTION
**Pull request by udayvunnam**
##### Summary:

Add a descriptive title for the dashboard

##### Jira Links:

(https://harness.atlassian.net/browse/CDS-38222)

##### Screenshots:

Screenshot 2022-05-30 at 10 35 32 AM

#### Next steps(optional)
create a utility to make sure each new route has a title, console warn in dev only.

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by udayvunnam** [comment]
retrigger titlecheck

File: src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.test.tsx [hunk 1]
```diff
@@ -82,6 +82,7 @@ describe('Landing Dashboard Page', () => {
     expect(getByText('projectsOrgs.landingDashboard.dashboardTitle')).toBeDefined()
     expect(getByText('common.welcome')).toBeDefined()
     expect(getByText('projectsOrgs.landingDashboard.atAGlance')).toBeDefined()
+    await waitFor(() => expect(document.title).toEqual('dashboardLabel | harness'))
```
Comment (kapilkumawat86) [inline src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.test.tsx:85]: how this is working, it should be equals to dashboardLabel

File: src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.tsx [hunk 2]
```diff
@@ -42,6 +43,7 @@ const LandingDashboardPage: React.FC = () => {
   const { accountId } = useParams<AccountPathProps>()
   const { currentUserInfo } = useAppStore()
   const { getString } = useStrings()
+  useDocumentTitle(getString('dashboardLabel'))
```
Comment (kapilkumawat86) [inline src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.tsx:46]: Do we need to run this on every render, Can we move to useEffect this no dependencies

Comment (udayvunnam) [reply src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.tsx:46]: doesn't useDocumentTitle internally uses useEffect(useDocumentTitle)?

**Review by reetika04 — APPROVED** [review]

File: src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.test.tsx (hunk 1 above)
Comment (udayvunnam) [reply src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.test.tsx:85]: TestWrapper has mocked `getString = (key: string) => key`, ideally we can try wrapping actual String Context Provider to get real value. Regarding `| harness` - useDocumentTitle adds that additional string

**Review by kapilkumawat86 — APPROVED** [review]

File: src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.tsx (hunk 2 above)
Comment (kapilkumawat86) [reply src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.tsx:46]: ok.. long time I didn't check that code

File: src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.test.tsx (hunk 1 above)
Comment (kapilkumawat86) [reply src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.test.tsx:85]: ok

File: src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.tsx (hunk 2 above)
Comment (ishantmehtaharness) [reply src/modules/45-projects-orgs/pages/LandingDashboardPage/LandingDashboardPage.tsx:46]: useDocumentTitle uses useDeepCompareEffect, which internally uses useEffect

wowww

**Pull request merged**